### PR TITLE
Added documentation note on definition generic parameters

### PR DESCRIPTION
**Pull request by ilyapuchka**
Look's like develop is a protected brunch now so everything should go through PRs, or is there another way?

### CODE REVIEW

**Comment by AliSoftware** [comment]
Oh wait, I protected `develop` earlier but I though you'd have the right to push to it anyway, sorry.
I'll remove the push protection on it and only protect master then.

**Comment by AliSoftware** [comment]
Ok I kept the anti-deletion + anti-force-push protection on develop, but removed the obligation to use PRs on it. Could you try directly pushing on `develop` again to check it's lifted now?

File: Dip/Dip/Definition.swift [hunk 1]
```diff
@@ -56,7 +56,12 @@ public enum ComponentScope {
   case Singleton
 }
 
-///Definition of type T describes how instances of this type should be created when they are resolved by container.
+/**
+ Definition of type T describes how instances of this type should be created when this type is resolved by container.
+ Generic parameter `T` is the type of the instance to resolve, `F` is the type of block-factory that creates an instance of T.
```
Comment (AliSoftware) [inline Dip/Dip/Definition.swift]: Using a bullet point list may be easier to read :wink:

**Comment by ilyapuchka** [comment]
Done, thanks! Closing this one.

**Pull request closed without merging**